src/app/profiles.rs

```diff
@@ -267,6 +274,15 @@ fn convert_route(
     Some((req_match, route))
 }
 
+fn convert_dst_override(orig: api::WeightedDst) -> Option<profiles::WeightedAddr> {
+    NameAddr::from_str(orig.authority.as_str())
+        .ok()
+        .map(|addr| profiles::WeightedAddr {
+            addr,
+            weight: orig.weight,
```

Comment:
Should we test that the weight is non-zero here?

Reply:
We discard zero-weighted items when constructing the weighed distribution, so I don't think we need to test that here as well.